requirements/travis.in

```diff
@@ -0,0 +1,6 @@
+-c constraints.txt
+
+-r test.txt
```

Comment:
I think that this is not needed here, all of that will be installed in the tox environment